layout.md

```diff
@@ -1,10 +1,13 @@
 ---
 layout: page
-title: Layouts in Flutter
+title: Dealing with box constraints in Flutter
 sidebar: home_sidebar
 permalink: /layout/
```

Comment:
Yeah that's probably a good idea. Let's do that in a separate set of PRs though, since it'll also involve changes to the framework and isn't required for improving the docs on the short term.